test/martha/martha_v3.test.js

```diff
@@ -883,19 +899,25 @@ test.serial('martha_v3 parses a Kids First CIB URI response correctly', async (t
     t.is(response.statusCode, 200);
 
     const result = response.send.lastCall.args[0];
-    sinon.assert.callCount(getJsonFromApiStub, 2); // Bond was called to get SA key
-    t.deepEqual({ ...result }, kidsFirstDrsMarthaResult(googleSAKeyObject));
+    sinon.assert.callCount(getJsonFromApiStub, 3); // DRS metadata, Bond access token, DRS access URL
+    t.deepEqual({ ...result }, kidsFirstDrsMarthaResult(drsAccessUrlResponse));
     t.is(
         getJsonFromApiStub.getCall(0).args[0],
-        `https://${config.HOST_KIDS_FIRST_STAGING}/ga4gh/dos/v1/dataobjects/ed6be7ab-068e-46c8-824a-f39cfbb885cc`,
+        `https://${config.HOST_KIDS_FIRST_STAGING}/ga4gh/drs/v1/objects/ed6be7ab-068e-46c8-824a-f39cfbb885cc`,
     );
     t.falsy(getJsonFromApiStub.getCall(0).args[1]); // no auth passed
-    const requestedBondUrl = getJsonFromApiStub.getCall(1).args[0];
-    const matches = requestedBondUrl.match(bondRegEx);
-    t.truthy(matches, 'Bond URL called does not match Bond URL regular expression');
-    const expectedProvider = 'dcf-fence';
-    const actualProvider = matches[2];
-    t.is(actualProvider, expectedProvider);
+    const requestedBondAccessToken = getJsonFromApiStub.getCall(1).args[0];
+    const accessTokenMatches = requestedBondAccessToken.match(bondAccessTokenRegEx);
+    t.truthy(accessTokenMatches, 'Bond URL called does not match Bond URL regular expression');
+    const expectedAccessTokenProvider = 'kids-first';
+    const actualAccessTokenProvider = accessTokenMatches[2];
+    t.is(actualAccessTokenProvider, expectedAccessTokenProvider);
+    t.is(
+        getJsonFromApiStub.getCall(2).args[0],
```

Comment:
Minor whitespace consistency again: add a space before and after the checks during `getCall(1)` to separate the section from `getCall(0)` and `getCall(2)`.